### DISCUSSION TITLE
'We are unable to complete your signup. Please read this guide for further details.'

### DISCUSSION THREAD

**Discussion by yousefeltrach** [vercel/community#5570] [Account Issue]
### Question

I created my account but I can't log in  they gave me this message  'We are unable to complete your signup. Please read this guide for further details.'

**Comment by amyegan** [comment]
Hi @yousefeltrach. If your account was already created, you should be able to log in at [vercel.com/login](https://vercel.com/login). Please make sure you are not trying to log in on the sign up page 🙏

**Reply by yousefeltrach** [reply]
no i am in the login page but it's not logged when i  write my email it says the message above

**Reply by yousefeltrach** [reply]
![sssssss](https://github.com/vercel/community/assets/118273934/10991d03-f18d-4e55-9d2e-188e153feb72)

**Reply by yousefeltrach** [reply]
@amyegan can you help

**Reply by Belloticyd** [reply]
i have the same issue with my account as @yousefeltrach issue with his account as well... please help me
